README.md

```diff
@@ -26,6 +26,22 @@
   </a>
 </p>
 
+### Overview
+
+Bespoke Curator makes it very easy to create high-quality synthetic data at scale, which you can use to finetune models or use for structured data extraction at scale.
+
+Bespoke Curator is an open-source project:
+* That comes with a rich Python based library for generating and curating synthetic data.
+* A Curator Viewer which makes it easy to view the datasets, thus aiding in the dataset creation.
+* We will also be releasing high-quality datasets that should move the needle on post-training.
+
+### Key Features
+
+1. **Programmability and Structured Outputs**: Synthetic data generation is lot more than just calling one prompt. It is a sequence of calling LLMs. You can orchestrate complex pipelines of calling LLMs and use structured output to decide on control-flow. Curator treats structured outputs as first class citizens.
```

Comment:
s/It is a sequence/It involves a sequence

Reply:
Updated